lib/squib/layout_parser.rb

```diff
@@ -76,6 +80,18 @@ def sub_parent_child(parent, child)
       parent_pixels - child_pixels
     end
 
+    def mul_parent_child(parent, child)
+      parent_pixels = Args::UnitConversion.parse(parent, @dpi).to_f
+      child_float = child.sub('x=', '').to_f
```

Comment:
no units on multipliers or divisors. I don't think someone will want ``x=10mm`` to silently multiply by some larger number based on dpi.